x-pack/auditbeat/module/system/socket/guess/deref.go

```diff
@@ -79,22 +79,22 @@ func (g *guessDeref) Requires() []string {
 
 // Probes returns a kprobe on uname() that dumps the first bytes
 // pointed to by its first parameter.
-func (g *guessDeref) Probes() ([]helper.ProbeDef, error) {
-	return []helper.ProbeDef{
+func (g *guessDeref) Probes() ([]kprobes.ProbeDef, error) {
+	return []kprobes.ProbeDef{
 		{
 			Probe: tracing.Probe{
```

Comment:
wondering why keep the package different for `kprobes` v. `tracing`? Are we planning on doing some other types of tracing (i.e. tracepoint, uprobe, etc?)

Reply:
Not really. Makes sense to merge the packages.